civis/tests/test_futures.py

```diff
@@ -33,6 +34,14 @@ def clear_lru_cache():
     generate_classes.cache_clear()
 
 
+def _create_poller_mock(state: str) -> mock.Mock:
+    poller = mock.Mock()
+    api_result = mock.Mock()
+    api_result.state = state
```

Comment:
not necessary but I think you can cut down on lines like this:
```
api_result = mock.Mock(state=state)
poller = mock.Mock(return_value=api_result)
```

Reply:
I dig it -- I'll update this!